src/shared/components/plots/MultipleCategoryBarPlot.tsx

```diff
@@ -917,3 +929,39 @@ export default class MultipleCategoryBarPlot extends React.Component<
         );
     }
 }
+
+type QPLabelProps = {
+    x: number,
+    y: number,
+    pValue: number | null,
+    qValue: number | null,
+};
+
+const PQValueLabel = (props: QPLabelProps) => {
+    const pFormatted = formatLabel('p', props.pValue);
+    const qFormatted= formatLabel('q', props.qValue)
+    return (
+        <foreignObject
+            x={props.x}
+            y={props.y}
+            width="100%"
+            height="3em"
+            style={{fontSize: "0.8em"}}
+        >
+            <g x={props.x} y={props.y}>
+                <div className="p-value-label">{pFormatted}</div>
+                <div className="q-value-label">{qFormatted}</div>
+            </g>
+        </foreignObject>
+    );
+
+    function formatLabel(name: string, value: number | null) {
+        if (value != null) {
+            return <>
+                {name} {toConditionalPrecisionWithMinimum(value, 3, 0.01, -10, true)}
```

Comment:
see comment below.  why wouldn't you put equal sign logic here?

Reply:
See https://github.com/cBioPortal/cbioportal-frontend/pull/4234#discussion_r871079157